commit 275c719e67ba136937d1416da9aaa6f0d37e85b7
Author: Miroslav Rezanina <mrezanin@redhat.com>
Date:   Wed Apr 3 08:41:32 2013 +0200

    rtc-test: Fix test failures with recent glib
    
    As of glib 2.35.4, glib changed its logic for ordering test cases:
    
    https://bugzilla.gnome.org/show_bug.cgi?id=694487
    
    This was causing failures in rtc-test. Group the reordered test
    cases into their own suite, which maintains the original ordering.
    
    Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

diff --git a/tests/rtc-test.c b/tests/rtc-test.c
index 203c0fc..097cfb4 100644
--- a/tests/rtc-test.c
+++ b/tests/rtc-test.c
@@ -384,11 +384,11 @@ int main(int argc, char **argv)
 
     qtest_add_func("/rtc/bcd/check-time", bcd_check_time);
     qtest_add_func("/rtc/dec/check-time", dec_check_time);
-    qtest_add_func("/rtc/alarm-time", alarm_time);
+    qtest_add_func("/rtc/alarm/alarm-time", alarm_time);
     qtest_add_func("/rtc/set-year/20xx", set_year_20xx);
     qtest_add_func("/rtc/set-year/1980", set_year_1980);
-    qtest_add_func("/rtc/register_b_set_flag", register_b_set_flag);
-    qtest_add_func("/rtc/fuzz-registers", fuzz_registers);
+    qtest_add_func("/rtc/misc/register_b_set_flag", register_b_set_flag);
+    qtest_add_func("/rtc/misc/fuzz-registers", fuzz_registers);
     ret = g_test_run();
 
     if (s) {
